section-3/exercises/denysKozlovskyiSection3.js

```diff
@@ -0,0 +1,108 @@
+// Exercise 1:
+// 1. Create an empty array numbers, then add values 10, 20, 30 using index positions.
+let numberArray  = [];
```

Comment:
Why you decelerate numberArray as let ? You didn't reassign numberArray in this exercise. Even if you add something to array it still should be **const**